### PR TITLE
Bugfix/fix issue with stacktrace

### PR DESCRIPTION
**Pull request by Vmak11**

### CODE REVIEW

**Comment by denis-sokolov** [comment]
Thank you for your contribution. This seems a useful addition.

File: src/Whoops/Run.php [hunk 1]
```diff
@@ -395,7 +395,7 @@ private function writeToOutputNow($output)
         return $this;
     }
 
-    private static function isLevelFatal($level)
+    public static function isLevelFatal($level)
```
Comment (GrahamCampbell) [inline src/Whoops/Run.php]: This feels dodgie to me.

Comment (GrahamCampbell) [inline src/Whoops/Run.php]: Maybe we should move this method somewhere.

Comment (Vmak11) [inline src/Whoops/Run.php]: Yes, agree, how about Whoops/Util/Misc.php?

**Pull request merged**